crates/swc/tests/errors/lints/no-empty-function/default/ts/output.swc-stderr

```diff
@@ -0,0 +1,30 @@
+error: Unexpected empty constructor pattern
+ 
+  |
+2 |   private constructor() {}
```

Comment:
I think `private consturctor`s are useful and we should not report an error for this.
Does `eslint` print an error for this?

Reply:
I can make option for this like

```
constructors: {
  allow: ["private"]
}
```

Reply:
Hmm... In my opinion, it should be allowed by default because it's a valid pattern and has a clear purpose.
I'm not sure about default of other linters, though

Reply:
It's really uncommon patten for me. If you think that it's should be ignored by default, I'll made it

Reply:
I found https://github.com/typescript-eslint/typescript-eslint/issues/1178 and hmm... I think error by default is fine, but I prefer to have an option to allow private constructors.

Reply:
Ok. I'll allow to turn off it